modules/rostests/apitests/user32/PrivateExtractIcons.c

```diff
@@ -110,11 +121,14 @@ static struct
     {L"%temp%\\cpimg2e.exe", IDR_EXE_NORMAL}
 };
 
+void TestPairExtraction(void);
+
```

Comment:
Could the new function itself be moved here (along with struct), so you wouldn't need forward declaration?